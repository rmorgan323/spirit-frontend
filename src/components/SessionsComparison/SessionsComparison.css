/******  GLOBAL CSS VARIABLES  ******/
/***  COLORS  ***/
/***  FONTS  ***/
* {
  font-size: 16px;
  font-family: "Lato", sans-serif;
  outline: none;
  color: #6d6e71; }

h1, h2, h3, h4, h5, h6 {
  font-family: "Josefin Sans", sans-serif;
  line-height: 1.3; }

h1 {
  font-size: 44px; }

h2 {
  font-size: 30px; }

h3 {
  font-size: 24px; }

h4 {
  font-size: 18px; }

h5 {
  font-size: 14px; }

h6 {
  font-size: 10px; }

p {
  font-size: 14px;
  line-height: 1.3;
  font-weight: 900; }

::placeholder {
  opacity: .5;
  text-align: center; }

/***  MIXINS  ***/
.SessionsComparison h3 {
  text-align: center; }

.SessionsComparison h4 {
  margin-bottom: 10px;
  text-align: center; }

.SessionsComparison .comparison-section-holder {
  width: 1040px;
  margin: auto;
  display: flex;
  flex-wrap: wrap;
  padding-bottom: 50px; }
