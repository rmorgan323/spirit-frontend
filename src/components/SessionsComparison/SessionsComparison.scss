@import '../../variables.scss';

.SessionsComparison {
  h3 {
    text-align: center;
  }

  h4 {
    margin-bottom: 10px;
    text-align: center;
  }

  .comparison-section-holder {
    width: 1040px;
    margin: auto;
    display: flex;
    flex-wrap: wrap;
    padding-bottom: 50px;
  }
}