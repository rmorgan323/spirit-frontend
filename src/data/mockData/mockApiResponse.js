const mockApiResponse = {
  status: 'success'
};

export default mockApiResponse;