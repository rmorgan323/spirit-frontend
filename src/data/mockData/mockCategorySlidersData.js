/*eslint-disable camelcase*/

export const mockTitle = 'Capacities';

export const mockSlidersArray = [
  'Attention',
  'Action',
  'Affect',
  'Autonomic',
  'Arousability'
];

export const mockDatabaseNamesArray = [
  'mod_2_attention',
  'mod_2_action',
  'mod_2_affect',
  'mod_2_autonomic',
  'mod_2_arousability'
];

export const mockReminderAsterisk = [null, '*', '*', '*', '*'];

export const mockReminder = '* If selected, remember to make a hypothesized arousal chart.';

export const mockSliderTitle = 'Attention';

export const mockDatabaseName = 'mod_2_attention';

