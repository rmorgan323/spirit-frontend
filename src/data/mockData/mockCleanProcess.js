/*eslint-disable camelcase*/

export const mockCleanProcess = {
  id: 10,
  sen_h_vestibular: null,
  sen_h_proprioception: null,
  sen_h_tactile: null,
  sen_h_auditory: null,
  sen_h_visual: null,
  sen_h_intero: null,
  sen_1_ability: null,
  sen_1_match: null,
  sen_1_classification: null,
  sen_1_spatial: null,
  sen_1_figure: null,
  sen_1_inferences: null,
  sen_2_initiation: null,
  sen_2_imitation: null,
  sen_2_affordance: null,
  sen_2_generativity: null,
  sen_3_sequencing: null,
  sen_3_sequencing_gestalt: null,
  sen_3_schema_selection: null,
  sen_3_schema_expansion: null,
  sen_3_modification: null,
  sen_4_timing: null,
  sen_4_timing_force: null,
  sen_4_spatial: null,
  sen_4_temporal: null,
  sen_4_control: null,
  sen_4_monitoring: null,
  sen_5_oral: null,
  sen_5_ocular: null,
  sen_5_postural: null,
  sen_5_limb: null,
  sen_5_gross: null,
  sen_5_fine: null,
  sen_5_tool: null,
  sen_5_projected: null,
  sen_6_somatodyspraxia: null,
  sen_6_visual: null,
  sen_6_generalized: null,
  sen_6_bmc: null,
  sen_6_praxis: null,
  mod_h_vestibular: null,
  mod_h_proprioception: null,
  mod_h_tactile: null,
  mod_h_auditory: null,
  mod_h_visual: null,
  mod_h_intero: null,
  mod_1_defensiveness: null,
  mod_1_over: null,
  mod_1_under: null,
  mod_1_variable: null,
  mod_2_attention: null,
  mod_2_action: null,
  mod_2_affect: null,
  mod_2_autonomic: null,
  mod_3_physiological: null,
  mod_3_sensory: null,
  mod_3_motor: null,
  mod_3_emotional: null,
  mod_3_cognitive: null,
  mod_3_social: null,
  mod_3_prosocial: null,
  mod_4a_passive: null,
  mod_4a_co_regulation: null,
  mod_4a_approach: null,
  mod_4a_adaptive: null,
  mod_4b_active: null,
  mod_4b_self_regulation: null,
  mod_4b_avoid: null,
  mod_4b_maladaptive: null,
  exe_1_orient: null,
  exe_1_inhibition: null,
  exe_2a_working: null,
  exe_2a_shifting: null,
  exe_2b_verbal: null,
  exe_2b_spatial: null,
  exe_2b_general: null,
  exe_2b_global: null,
  exe_2b_flexibility: null,
  exe_3_sequencing: null,
  exe_3_monitoring: null,
  exe_4a_future: null,
  exe_4a_spatial: null,
  exe_4a_problem: null,
  exe_4a_reflective: null,
  exe_4a_internalized: null,
  exe_4b_motivational: null,
  exe_4b_effortful: null,
  exe_4b_self_control: null,
  exe_4b_self_regulation: null,
  exe_4b_self_compassion: null,
  pos_h_vestibular: null,
  pos_h_proprioception: null,
  pos_h_tactile: null,
  pos_h_auditory: null,
  pos_h_visual: null,
  pos_h_intero: null,
  pos_2_stability: null,
  pos_2_mobility: null,
  pos_2_tonic: null,
  pos_2_phasic: null,
  pos_2_muscle: null,
  pos_3_sagittal: null,
  pos_3_frontal: null,
  pos_3_transverse: null,
  pos_3_midlines: null,
  pos_5_suck: null,
  pos_5_ocular: null,
  pos_5_antigravity: null,
  pos_5_reflex: null,
  pos_5_alignment_COG: null,
  pos_5_alignment_orient: null,
  pos_5_anticipatory: null,
  pos_5_interactive: null,
  pos_6_push: null,
  pos_6_bilateral: null,
  pos_6_grasp: null,
  pos_6_strength: null,
  pos_6_effortful: null,
  soc_1_regulation: null,
  soc_1_engagement: null,
  soc_1_reciprocity: null,
  soc_1_problem: null,
  soc_1_representation: null,
  soc_1_triangular: null,
  soc_1_comparative: null,
  soc_1_reflective: null,
  soc_2_self_awareness: null,
  soc_2_joint: null,
  soc_2_social_referencing: null,
  soc_2_imitation: null,
  soc_2_inferential: null,
  soc_2_contingent: null,
  soc_2_theory: null,
  soc_2_central: null,
  soc_2_mentalizing: null,
  soc_2_empathy: null,
  soc_2_social_motivators: null,
  soc_2_pro_social: null,
  soc_3_affective: null,
  soc_3_emotional_contagion: null,
  soc_3_activation_low: null,
  soc_3_emotional_cascade: null,
  soc_3_activation_high: null,
  soc_3_emotional_differentiation: null,
  soc_3_emotional_association: null,
  soc_3_self_control: null,
  soc_3_self_regulation: null,
  soc_3_self_compassion: null,
  soc_4_mental: null,
  soc_4_thoughts: null,
  soc_4_attachments: null,
  soc_4_trauma: null,
  session_id: 10,
  created_at: "2018-02-20T21:25:45.454Z",
  updated_at: "2018-02-20T21:25:45.454Z",
  mod_2_arousability: null,
  exe_3_initiation: null,
  exe_3_planning: null,
  pos_4_core: null,
  pos_4_shoulder: null,
  pos_4_pelvic: null,
  pos_4_head: null,
  pos_4_eyes: null,
  pos_4_hand: null,
  pos_4_lower: null,
  pos_4_foot: null,
  soc_2_social_perspective: null,
  pos_1_outer: null,
  pos_1_inner: null,
  exe_4a_generativity: null
};
