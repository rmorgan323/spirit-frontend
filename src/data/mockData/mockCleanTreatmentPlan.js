/*eslint-disable camelcase*/

export const mockCleanTreatmentPlan = {
  id: 10,
  modulation_sensory: null,
  modulation_task: null,
  modulation_environment: null,
  modulation_predictability: null,
  modulation_self_regulation: null,
  modulation_interaction: null,
  modulation_JRC_AR_notes: null,
  posture_sensory: null,
  posture_task: null,
  posture_environment: null,
  posture_predictability: null,
  posture_self_regulation: null,
  posture_interaction: null,
  posture_JRC_AR_notes: null,
  sensory_discrimination_sensory: null,
  sensory_discrimination_task: null,
  sensory_discrimination_environment: null,
  sensory_discrimination_predictability: null,
  sensory_discrimination_self_regulation: null,
  sensory_discrimination_interaction: null,
  sensory_discrimination_JRC_AR_notes: null,
  social_emotional_sensory: null,
  social_emotional_task: null,
  social_emotional_environment: null,
  social_emotional_predictability: null,
  social_emotional_self_regulation: null,
  social_emotional_interaction: null,
  social_emotional_JRC_AR_notes: null,
  executive_functioning_sensory: null,
  executive_functioning_task: null,
  executive_functioning_environment: null,
  executive_functioning_predictability: null,
  executive_functioning_self_regulation: null,
  executive_functioning_interaction: null,
  executive_functioning_JRC_AR_notes: null,
  session_id: 10,
  created_at: "2018-02-20T21:25:45.468Z",
  updated_at: "2018-02-20T21:25:45.468Z"
};
