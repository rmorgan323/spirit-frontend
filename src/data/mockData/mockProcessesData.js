/*eslint-disable camelcase*/

export const mockProcessesData = {
  executive: [
    {
      dbName: 'exe_1_orient',
      title: 'Orient EYES-EARS-SELF to GOAL',
      values: ['9A', '10F']
    }
  ],
  modulation: [
    {
      dbName: 'mod_h_vestibular',
      title: 'Vestibular',
      values: [null, '7I']
    }
  ],
  postural: [
    {
      dbName: 'pos_h_tactile',
      title: 'Tactile',
      values: ['2I', '7A']
    }
  ],
  sensory: [
    {
      dbName: 'sen_1_figure',
      title: 'Figure Ground Perception',
      values: ['6I', '8I']
    }
  ],
  social: [
    {
      dbName: 'soc_2_joing',
      title: 'Joint Attention',
      values: ['1R', '5I']
    }
  ]
};
