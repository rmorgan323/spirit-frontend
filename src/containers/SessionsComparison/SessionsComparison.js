import React from 'react';
import { connect } from 'react-redux';
import { PropTypes } from 'prop-types';
import ComparisonCard from '../../components/ComparisonCard/ComparisonCard';
import './SessionsComparison.css';

export const SessionsComparison = props => {
  let modulationDisplay;
  let posturalDisplay;
  let sensoryDisplay;
  let socialDisplay;
  let executiveDisplay;

  if (Object.keys(props.processesData).length) {
    const dateArray = props.processesData.modulation[0].values;

    modulationDisplay = props.processesData.modulation.map((object, index) => {
      if (index > 0) {
        return (
          <ComparisonCard
            dateArray={dateArray}
            title={object.title}
            values={object.values}
          />
        );
      }
    });

    posturalDisplay = props.processesData.postural.map((object, index) => {
      if (index > 0) {
        return (
          <ComparisonCard
            dateArray={dateArray}
            title={object.title}
            values={object.values}
          />
        );
      }
    });

    sensoryDisplay = props.processesData.sensory.map((object, index) => {
      if (index > 0) {
        return (
          <ComparisonCard
            dateArray={dateArray}
            title={object.title}
            values={object.values}
          />
        );
      }
    });

    socialDisplay = props.processesData.social.map((object, index) => {
      if (index > 0) {
        return (
          <ComparisonCard
            dateArray={dateArray}
            title={object.title}
            values={object.values}
          />
        );
      }
    });

    executiveDisplay = props.processesData.executive.map((object, index) => {
      if (index > 0) {
        return (
          <ComparisonCard
            dateArray={dateArray}
            title={object.title}
            values={object.values}
          />
        );
      }
    });
  }

  return (
    <div className="SessionsComparison">
      <h3>Sessions Comparison</h3>

      <h4>Modulation</h4>
      <div className="comparison-section-holder">{modulationDisplay}</div>

      <h4>Postural</h4>
      <div className="comparison-section-holder">{posturalDisplay}</div>

      <h4>Sensory</h4>
      <div className="comparison-section-holder">{sensoryDisplay}</div>

      <h4>Social</h4>
      <div className="comparison-section-holder">{socialDisplay}</div>

      <h4>Executive</h4>
      <div className="comparison-section-holder">{executiveDisplay}</div>
    </div>
  );
};

export const mapStateToProps = store => ({
  processesData: store.processesData
});

export default connect(mapStateToProps, null)(SessionsComparison);

SessionsComparison.propTypes = {
  processesData: PropTypes.object
};
