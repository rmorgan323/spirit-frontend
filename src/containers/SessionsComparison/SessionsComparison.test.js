import React from 'react';
import { SessionsComparison, mapStateToProps } from './SessionsComparison';
import { shallow } from 'enzyme';
import { mockProcessesData } from '../../data/mockData/mockProcessesData';

describe('SessionsComparison tests', () => {
  let processesData;
  let renderedSessionsComparison;

  beforeEach(() => {
    processesData = mockProcessesData;

    renderedSessionsComparison = shallow(
      <SessionsComparison processesData={processesData} />
    );
  });

  it('Should match the snapshot', () => {
    expect(renderedSessionsComparison).toMatchSnapshot();
  });
});

describe('mapStateToProps tests', () => {
  it('Should pull processesData from store', () => {
    const processesData = mockProcessesData;
    const mockStore = { processesData };
    const result = mapStateToProps(mockStore);

    expect(result.processesData).toEqual(mockStore.processesData);
  });
});
