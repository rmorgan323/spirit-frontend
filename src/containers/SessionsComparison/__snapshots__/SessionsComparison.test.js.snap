// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SessionsComparison tests Should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown
    processesData={
      Object {
        "executive": Array [
          Object {
            "dbName": "exe_1_orient",
            "title": "Orient EYES-EARS-SELF to GOAL",
            "values": Array [
              "9A",
              "10F",
            ],
          },
        ],
        "modulation": Array [
          Object {
            "dbName": "mod_h_vestibular",
            "title": "Vestibular",
            "values": Array [
              null,
              "7I",
            ],
          },
        ],
        "postural": Array [
          Object {
            "dbName": "pos_h_tactile",
            "title": "Tactile",
            "values": Array [
              "2I",
              "7A",
            ],
          },
        ],
        "sensory": Array [
          Object {
            "dbName": "sen_1_figure",
            "title": "Figure Ground Perception",
            "values": Array [
              "6I",
              "8I",
            ],
          },
        ],
        "social": Array [
          Object {
            "dbName": "soc_2_joing",
            "title": "Joint Attention",
            "values": Array [
              "1R",
              "5I",
            ],
          },
        ],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Sessions Comparison
        </h3>,
        <h4>
          Modulation
        </h4>,
        <div
          className="comparison-section-holder"
        />,
        <h4>
          Postural
        </h4>,
        <div
          className="comparison-section-holder"
        />,
        <h4>
          Sensory
        </h4>,
        <div
          className="comparison-section-holder"
        />,
        <h4>
          Social
        </h4>,
        <div
          className="comparison-section-holder"
        />,
        <h4>
          Executive
        </h4>,
        <div
          className="comparison-section-holder"
        />,
      ],
      "className": "SessionsComparison",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sessions Comparison",
        },
        "ref": null,
        "rendered": "Sessions Comparison",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Modulation",
        },
        "ref": null,
        "rendered": "Modulation",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
          ],
          "className": "comparison-section-holder",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Postural",
        },
        "ref": null,
        "rendered": "Postural",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
          ],
          "className": "comparison-section-holder",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Sensory",
        },
        "ref": null,
        "rendered": "Sensory",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
          ],
          "className": "comparison-section-holder",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Social",
        },
        "ref": null,
        "rendered": "Social",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
          ],
          "className": "comparison-section-holder",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Executive",
        },
        "ref": null,
        "rendered": "Executive",
        "type": "h4",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
          ],
          "className": "comparison-section-holder",
        },
        "ref": null,
        "rendered": Array [
          undefined,
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Sessions Comparison
          </h3>,
          <h4>
            Modulation
          </h4>,
          <div
            className="comparison-section-holder"
          />,
          <h4>
            Postural
          </h4>,
          <div
            className="comparison-section-holder"
          />,
          <h4>
            Sensory
          </h4>,
          <div
            className="comparison-section-holder"
          />,
          <h4>
            Social
          </h4>,
          <div
            className="comparison-section-holder"
          />,
          <h4>
            Executive
          </h4>,
          <div
            className="comparison-section-holder"
          />,
        ],
        "className": "SessionsComparison",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sessions Comparison",
          },
          "ref": null,
          "rendered": "Sessions Comparison",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Modulation",
          },
          "ref": null,
          "rendered": "Modulation",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
            ],
            "className": "comparison-section-holder",
          },
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Postural",
          },
          "ref": null,
          "rendered": "Postural",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
            ],
            "className": "comparison-section-holder",
          },
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sensory",
          },
          "ref": null,
          "rendered": "Sensory",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
            ],
            "className": "comparison-section-holder",
          },
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Social",
          },
          "ref": null,
          "rendered": "Social",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
            ],
            "className": "comparison-section-holder",
          },
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Executive",
          },
          "ref": null,
          "rendered": "Executive",
          "type": "h4",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
            ],
            "className": "comparison-section-holder",
          },
          "ref": null,
          "rendered": Array [
            undefined,
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
